src/activities/Calender/Calender.qml

```diff
@@ -0,0 +1,183 @@
+import "Calender.js" as Activity
+import "calender_dataset.js" as Dataset
+
+ActivityBase {
+    id: activity
+    property var dataset: Dataset
+    onStart: focus = true
+    onStop: {}
+
+    pageComponent: Image {
+        id: background
+        signal start
+        signal stop
+        fillMode: Image.PreserveAspectCrop
+        source: "qrc:/gcompris/src/activities/fifteen/resource/background.svg"
+        sourceSize.width: Math.max(parent.width, parent.height)
+
+        Component.onCompleted: {
+            activity.start.connect(start)
+            activity.stop.connect(stop)
+        }
+
+        // Add here the QML items you need to access in javascript
+        QtObject {
+            id: items
+            property Item main: activity.main
+            property alias background: background
+            property alias bar: bar
+            property alias bonus: bonus
+            property alias calender: calender
+            property alias okButton: okButton
+            property alias questionItemBackground: questionItemBackground
+            property alias questionItem: questionItem
+            property alias questionsModel: questionsModel
+            property alias score: score
+
+        }
+
+        onStart: { Activity.start(items, dataset) }
+        onStop: { Activity.stop() }
+
+        Rectangle{
+            id: calenderBox
+            width: parent.width * 0.40
+            height: parent.width * 0.40
+            anchors.bottom: bar.top
+            anchors.horizontalCenter: parent.horizontalCenter
+            color: "black"
+            opacity: 0.3
+
+        }
+        Calendar {
+            id: calender
+            weekNumbersVisible: false
+            width: calenderBox.width * 0.85
+            height: calenderBox.height * 0.85
+            anchors.centerIn: calenderBox
+            frameVisible: true
+            focus: true
+            onClicked:{
+                console.log(typeof selectedDate.getDate())
+                Activity.dateSelected = selectedDate
+            }
+        }
+
+        ListModel {
+            id: questionsModel
+        }
+
+        Rectangle {
+            id: questionItemBackground
+            color: "black"
+            border.width: 2
+            radius: 10
+            opacity: 0.85
+            z: 10
+            anchors{
+                horizontalCenter: parent.horizontalCenter
+                bottomMargin: 10
+            }
+            width: calenderBox.width * 2
+            height: calenderBox.height * 0.125
+            gradient: Gradient {
+                GradientStop { position: 0.0; color: "#000" }
+                GradientStop { position: 0.9; color: "#666" }
+                GradientStop { position: 1.0; color: "#AAA" }
+            }
+        }
+
+        GCText {
+            id: questionItem
+            text: "The date is: " + new Date().toLocaleDateString(Qt.locale("de_DE")) + " \n  " +new Date().toLocaleDateString(Qt.locale("en_US"))
```

Comment:
qsTr() for translatable text. .arg() for arguments
locale should be gotten from ApplicationSettings.locale